src/components/authorization/policies/by-role/consultant-manager.policy.ts

```diff
@@ -26,6 +26,14 @@ import { member, Policy, Role, sensMediumOrLower, sensOnlyLow } from '../util';
   r.Partnership.read.specifically((p) => [
     p.many('partner', 'organization').whenAny(member, sensOnlyLow).read,
   ]),
+  r.ProgressReport.whenAny(member, sensMediumOrLower).specifically(
+    (p) => p.status.read,
+  ),
+  [
+    r.ProgressReportCommunityStory,
+    r.ProgressReportHighlight,
+    r.ProgressReportTeamNews,
+  ].flatMap((it) => [it.read.specifically((p) => [p.responses.read])]),
```

Comment:
I believe the call out for _read_ perm on `responses` is redundant as that's the object has _read_ perm as well.
```suggestion
  ].flatMap((it) => [it.read]),
```

Reply:
I believe we do need this because of the way it is nested. Otherwise, the sections show but the responses themselves don't. Bryan and I paired on this yesterday afternoon.

Reply:
![I Dont Believe You Will Ferrell - 6JB4v4xPTAQFi](https://github.com/SeedCompany/cord-api-v3/assets/932566/cc3b0194-0aed-4e2f-813a-8513bb380e66)